### ISSUE TITLE
[Feature request]: Add mpris support

### ISSUE THREAD

**Issue by luigino** [cmdrkotori/mpc-qt#196]
So that the app can be controlled via qdbus.

It's not essential, of course, but most multimedia  players on linux seem to have it.

**Comment by ahjolinna** [comment]
There's a initial support already its just disable by default in the settings

**Comment by luigino** [comment]
@ahjolinna is org.mpris.MediaPlayer2.Player.OpenUri supported, to your knowledge?
Because I couldn't make it work.

**Comment by ahjolinna** [comment]
no, and its really basic support: https://github.com/cmdrkotori/mpc-qt/blob/master/DOCS/ipc.md#mpris

**Comment by ahjolinna** [comment]
there are some mpris features that don't work with kdeconnect (or was it vice versa?), but maybe @cmdrkotori  will add support later
